lib/integrations/externalauditstorage/bootstrap.go

```diff
@@ -200,8 +200,8 @@ func createTransientBucket(ctx context.Context, clt BootstrapS3Client, bucketNam
 					Expiration: &s3types.LifecycleExpiration{
 						Days: aws.Int32(1),
 					},
-					Filter: &s3types.LifecycleRuleFilterMemberPrefix{
-						Value: "/query_results",
+					Filter: &s3types.LifecycleRuleFilter{
+						Prefix: aws.String("/query_results"),
```

Comment:
I'm guessing here, although it does seem reasonable. @nklaassen, mind taking a look?

Reply:
looks correct to me